scripts/csi_test/csi_secrets

```diff
@@ -63,3 +63,9 @@ NodeUnstageVolumeSecret:
  username: USERNAME
  password: PASSWORD
  alias_in_storageclass: arr1
+
+ControllerPublishVolumeSecret:
```

Comment:
what about ControllerUnpublishVolume?

Reply:
adding ControllerUnpublishVolume was not defined as part of this ticket, if adding ControllerUnpublishVolume will require minor changes I will add it as part of this task otherwise I will open a new ticket

Reply:
ok. btw I think the same goes for two "volume lifecycle" tests in "Controller Service"

Reply:
"ControllerUnpublishVolume" test was added, for "volume lifecycle" tests a ticket was opened

Reply:
thanks for the addition and for opening CSI-2155!